rust/deny-check/Dockerfile

```
@@ -0,0 +1,17 @@
+#FROM rust:alpine
+FROM rust:slim
+
+#ENV GITHUB_WORKSPACE=""
+
+COPY deny.toml /deny.toml
+COPY start.sh /start.sh
+
+RUN apt update && apt upgrade -y && \
+    apt install -y libfindbin-libs-perl libssl-dev make && \
+    cargo install cargo-deny --no-default-features
+
+#RUN apk update --no-cache && apk upgrade --no-cache && \
+#    apk add --no-cache musl-dev openssl-dev openssl-libs-static pkgconf zlib-dev perl make && \
+#    cargo install cargo-deny --no-default-features
+
+CMD ["/start.sh"]
```

```suggestion
FROM rust:slim

COPY deny.toml /deny.toml
COPY start.sh /start.sh

RUN apt update && apt upgrade -y && \
    apt install -y libfindbin-libs-perl libssl-dev make && \
    cargo install cargo-deny --no-default-features

CMD ["/start.sh"]
```

I assume comments were left in from experiments and intended to be removed.

Two other questions:
1. What's `libfindbin-libs-perl` used for?
2. Do we need to `apt upgrade`? I imagine that slows down builds a bit.

---

1. `libfindbin-libs-perl` is required for the installation of cargo-deny. I forget which stage/dep requires it during compilation, but it *is* required.
2. `apt upgrade` isn't necessarily necessary. I was thinking GitHub would cache stages of building the image and I like having up-to-date software. I can remove that, though.